src/main.js

```diff
@@ -122,4 +123,12 @@ function displaySavedCovers() {
     </section>`;
   }
 }
+function removeCover() {
+  for (var i = 0; i < savedCovers.length; i++) {
```

Comment:
This function iterates through the entire array, regardless of whether the saved cover in question is found before reaching the last element in the array. How might you prevent that?

Reply:
by adding an if condition. So it wouldn't start iterating unless the condition is satisfied.

Reply:
I don’t think you need another if statement. Look into how you might be able to _break_ out of a loop